docker-for-aws/index.md

```diff
@@ -40,13 +40,10 @@ using CloudFormation. For more about stable and edge channels, see the
   </td>
   <td width="50%">
   {{aws_blue_edge}}
+  {{aws_blue_vpc_edge}}
   </td>
   </tr>
   <tr valign="top">
```

Comment:
The `<tr>` has two `<td>` in it just like it's supposed to. I think it's right. Both buttons are in one `<td>`.